core/dictgen/src/rootcling_impl.cxx

```diff
@@ -4485,9 +4485,11 @@ int RootClingMain(int argc,
    ROOT::TMetaUtils::TClingLookupHelper helper(interp, normCtxt, nullptr, nullptr, nullptr, nullptr);
    TClassEdit::Init(&helper);
 
-   // flags used only for the pragma parser:
-   clingArgs.push_back("-D__CINT__");
-   clingArgs.push_back("-D__MAKECINT__");
+   // flags used only for the pragma parser, for compatibility:
+   clingArgs.push_back("-D__CINT__"); // bw compatibility
+   clingArgs.push_back("-D__MAKECINT__"); // bw compatibility
+   clingArgs.push_back("-D__ICLING__");
+   clingArgs.push_back("-D__MAKECLING__");
```

Comment:
What is the semantic of these changes?   There is already 'new' spelling for those 2 (`__CLING__` and `__ROOTCLING__`) so adding the 2 new spelling does not seem to add much.  What am I missing?